great_expectations/checkpoint/checkpoint.py

```diff
@@ -8,6 +8,7 @@
 from uuid import UUID
 
 import great_expectations.exceptions as gx_exceptions
+from docs.sphinx_api_docs_source.tasks import public_api
```

Comment:
This is admittedly confusing - the decorators are in the _docs_decorators.py file
```suggestion
from great_expectations.core._docs_decorators import public_api
```

Reply:
I think this might also be causing the circular import issue in the deploy which is leading the api docs to not get built: https://app.netlify.com/sites/niobium-lead-7998/deploys/63c9ad1cd2a1f00008983912#L665 (why the deploy preview is not showing api docs).